packages/dcos-launch/extra/launch/aws.py

```diff
@@ -50,6 +48,8 @@ def zen_helper(self, config):
         will be provided (must be done in correct order) and added to the info
         JSON as 'temp_resources'
         """
+        if config['zen_helper'] != 'true':
+            return {}
```

Comment:
I don't see why an error would have much pertinence here... The point of this utility is: "If `key_helper: true` in the config YAML file then add a key. Otherwise, do nothing. Doing nothing is the intended default case.